lib/challenge.js

```diff
@@ -61,6 +91,24 @@ var some_pet;
 var Pet;
 // Answer Starts Here
 
+var Pet = function Pet(name, species, breed, noise) {
+  this.name = name;
+  this.species = species;
+  this.breed = breed;
+  this.noise = []];
+
+};
+
+Pet.prototype.makeNoise = function makeNoise () {
+  var makeNoises = 0;
```

Comment:
This choice of name is very dangerous. It's a one-character difference from the name of the function!